packages/web-twig/src/Twig/SvgExtension.php

```diff
@@ -195,11 +197,11 @@ private function makeReusableSVG(string $iconId, SimpleXMLElement $regularSvg)
 
         if ($attributes !== null && $reuseSvg instanceof SimpleXMLElement) {
             foreach ($attributes as $key => $value) {
-                if ($key !== self::ATTR_ID) {
-                    $reuseSvg->addAttribute((string) $key, (string) $value);
-                } else {
-                    $reuseSvg->addChild('use')->addAttribute('href', '#' . $iconId);
-                }
+              if ($key === self::ATTR_ID) {
```

Comment:
SvgExtensionTest.php is not reflected this chnages?

Reply:
It is. I changed the test_reusable.svg.